extension/scripts/features/stats-estimate/ttStatsEstimateEnemies.js

```diff
@@ -19,30 +19,36 @@
 		}
 	);
 
-	function registerListeners() {
-		addXHRListener(async ({ detail: { page, xhr } }) => {
-			if (!feature.enabled()) return;
-			if (page !== "userlist") return;
-
-			const step = new URLSearchParams(xhr.requestBody).get("step");
-			if (step !== "blackList") return;
-
-			new MutationObserver((mutations, observer) => {
-				showEstimates();
-				observer.disconnect();
-			}).observe(document.find(".blacklist"), { childList: true });
+	async function registerListeners() {
+		const listObserver = new MutationObserver((mutations) => {
+			if (mutations.some((mutation) => [...mutation.addedNodes].some((node) => node.matches("li[class*='tableRow__']")))) {
+				if (feature.enabled())
+					showEstimates();
+			}
 		});
+
+		const tableObserver = new MutationObserver((mutations) => {
+			if (mutations.some((mutation) => [...mutation.addedNodes].some((node) => node.tagName === "UL"))) {
+				if (feature.enabled()) {
+					showEstimates();
+					listObserver.observe(document.find(".tableWrapper > ul"));

```

Comment:
Uncaught TypeError: Failed to execute 'observe' on 'MutationObserver': The options object must set at least one of 'attributes', 'characterData', or 'childList' to true.